js/views/userCard.js

```diff
@@ -4,19 +4,38 @@ import loadTemplate from '../utils/loadTemplate';
 import app from '../app';
 import { followedByYou, followUnfollow } from '../utils/follow';
 import Profile from '../models/profile/Profile';
+import UserCard from '../models/UserCard';
 
 export default class extends BaseVw {
   constructor(options = {}) {
     super(options);
     this.options = options;
-    this.guid = options.guid || this.model.get('guid');
+
+    if (this.model instanceof Profile) {
+      this.guid = this.model.id;
+      this.fetched = true;
+    } else if (this.model) {
+      this.guid = this.model.get('guid');
+      this.fetched = false;
+    } else {
+      this.model = new UserCard({ guid: options.guid });
```

Comment:
Since all the userCard model does is give a guid, I think we could simplify the userCard view to either accept a guid or a profile model. If the creator of the userCard view has a userCard model, then it could just pass in the guid.